scripts/components/Repo.js

```diff
@@ -1,19 +1,24 @@
 'use strict';
 
 import React, { PropTypes } from 'react';
-import PureRenderMixin from 'react/lib/ReactComponentWithPureRenderMixin';
+import shallowEqual from 'react/lib/shallowEqual';
 import { Link } from 'react-router';
 
-const Repo = React.createClass({
-  mixins: [PureRenderMixin],
+class Repo extends React.Component {
 
-  propTypes: {
-    repo: PropTypes.object.isRequired,
-    owner: PropTypes.object.isRequired
-  },
+  // TODO: make it either a HigherOrderComponent
```

Comment:
I remember Sebastian saying they might have something else coming in the future to avoid subclassing even for pure components. I don't know what he meant though. For now, I'd like to choose to avoid any complication because it's not the point of this example.